.github/workflows/package.yml

```diff
@@ -10,14 +10,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
       fail-fast: false
     env:
       TZ: America/New_York
 
+    defaults:
```

Comment:
I like how you were able to clean up a lot of configs just by adding this default!